app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,11 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  {{#if canViewSisId}}
+  <a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}})
+  {{else}}
+  <a href="{{html_url}}" class="roster_user_name">{{short_name}}</a>
```

Comment:
I am not sure how display_name (shortname) can ever be blank but I will cover that.